src/data/complex/basic.lean

```diff
@@ -644,6 +644,9 @@ by rw [lt_def, not_and_distrib, not_lt]
 lemma not_le_zero_iff {z : ℂ} : ¬z ≤ 0 ↔ 0 < z.re ∨ z.im ≠ 0 := not_le_iff
 lemma not_lt_zero_iff {z : ℂ} : ¬z < 0 ↔ 0 ≤ z.re ∨ z.im ≠ 0 := not_lt_iff
 
+lemma eq_coe_re_of_real_le {r : ℝ} {z : ℂ} (hz : (r : ℂ) ≤ z) : z = z.re :=
```

Comment:
```
lemma eq_re_of_real_le {r : ℝ} {z : ℂ} (hz : (r : ℂ) ≤ z) : z = z.re :=
```
I think it is fine to drop `coe` from the name, we already do that in other places (e.g [is_R_or_C.re_eq_self_of_le](https://leanprover-community.github.io/mathlib_docs/data/complex/is_R_or_C.html#is_R_or_C.re_eq_self_of_le))